spec/src/main/asciidoc/metrics.asciidoc

```diff
@@ -28,6 +28,8 @@ the methods annotated with a `@Retry`, `@Timeout`, `@CircuitBreaker` or `@Bulkhe
 The automatically added metrics follow a consistent pattern which includes the fully qualified name of the annotated method.
 In the tables below, the placeholder `<name>` should be replaced by the fully qualified method name.
 
+If two methods have the same fully qualified name then the metrics for those methods will be combined.
```

Comment:
I would at least explain that the end result is not portable and may vary across implementations. For portable behavior, monitored methods in the same class should have unique names.

Reply:
Makes sense, I will add that.

Reply:
Done.